docker-compose.dev.yml

```
@@ -0,0 +1,65 @@
+services:
+  web:
+    container_name: web
+    image: ghcr.io/vim/website_next_generation/web:${TAG:-development}
+    environment:
+      - CMS_API=http://cms:1337/api
+      - NEXTAUTH_SECRET=${NEXTAUTH_SECRET}
+    ports:
+      - 3000:3000
+    networks:
+      - vim_network
+
+  db:
+    container_name: db
+    image: mysql:8
+    restart: unless-stopped
+    environment:
+      - MYSQL_ROOT_PASSWORD=${DATABASE_PASSWORD}
+      - MYSQL_USER=${DATABASE_USERNAME}
+      - MYSQL_PASSWORD=${DATABASE_PASSWORD}
+      - MYSQL_DATABASE=${CMS_DATABASE_NAME}
+    volumes:
+      - vim-cms-data:/var/lib/mysql
+      - ./db/docker-scripts:/docker-entrypoint-initdb.d
+    ports:
+      - ${DATABASE_PORT}:3306
```

Do we need this in a variable?
If so we could also use it twice like so:
```suggestion
      - ${DATABASE_PORT}:${DATABASE_PORT}
```

---

If the VM does not have anything running on 3306 anymore, we could remove the value and hardcode it.
But I wouldn't use the variable twice. The right hand side of the port is for specifying the internal container port.
Changing it would also require to configure mysql to run on a different port, which does not serve any purpose.
I'd leave it as is, as sometimes it might be helpful to be able to change the port with a single command like this

`DATABASE_PORT=3307 docker compose -f docker-composer.dev.yml up -d `